etc/meteo-qt.profile

```diff
@@ -0,0 +1,43 @@
+# Firejail profile for meteo-qt
+# This file is overwritten after every install/update
+# Persistent local customizations
+include meteo-qt.local
+# Persistent global definitions
+include globals.local
+
```

Comment:
For consistency with existing profiles, add a comment:
`# Allow python (blacklisted by disable-interpreters.inc)`